backend/v3/api/org/v2/org.go

```diff
@@ -31,3 +48,137 @@ package orgv2
 // 	}
 // 	return cmds
 // }
+
+func UpdateOrganization(ctx context.Context, request *connect.Request[v2beta_org.UpdateOrganizationRequest]) (*connect.Response[v2beta_org.UpdateOrganizationResponse], error) {
+	orgUpdtCmd := domain.NewUpdateOrgCommand(request.Msg.GetId(), request.Msg.GetName())
```

Comment:
I think we can add these two extra letters :D
```suggestion
	orgUpdateCmd := domain.NewUpdateOrgCommand(request.Msg.GetId(), request.Msg.GetName())
```